LayoutKit/Alignment.swift

```diff
@@ -117,11 +117,11 @@ public struct Alignment {
     }
 
     /// A function that aligns size in rect.
-    public typealias Aligner = (size: CGSize, rect: CGRect) -> CGRect
+    public typealias Aligner = (_ size: CGSize, _ rect: CGRect) -> CGRect
```

Comment:
https://github.com/apple/swift-evolution/blob/master/proposals/0111-remove-arg-label-type-significance.md

Yeah looks like this kind of pattern is killed in Swift 3 😢